src/processTailwindFeatures.js

```diff
@@ -12,18 +11,16 @@ import generateUtilities from './util/generateUtilities'
 import processPlugins from './util/processPlugins'
 
 export default function(getConfig) {
-  return function(css) {
-    const config = getConfig()
-    const processedPlugins = processPlugins(config)
-    const utilities = generateUtilities(config, processedPlugins.utilities)
+  const config = getConfig()
+  const processedPlugins = processPlugins(config)
+  const utilities = generateUtilities(config, processedPlugins.utilities)
 
-    return postcss([
-      substituteTailwindAtRules(config, processedPlugins, utilities),
-      evaluateTailwindFunctions(config),
-      substituteVariantsAtRules(config, processedPlugins),
-      substituteResponsiveAtRules(config),
-      substituteScreenAtRules(config),
-      substituteClassApplyAtRules(config, utilities),
-    ]).process(css, { from: _.get(css, 'source.input.file') })
-  }
+  return postcss([
+    substituteTailwindAtRules(config, processedPlugins, utilities),
+    evaluateTailwindFunctions(config),
+    substituteVariantsAtRules(config, processedPlugins),
+    substituteResponsiveAtRules(config),
+    substituteScreenAtRules(config),
+    substituteClassApplyAtRules(config, utilities),
+  ])
 }
```

Comment:
So if I remember correctly it's actually important to keep this as it was otherwise Webpack won't properly respond to changes in the config file in watch mode:

https://github.com/tailwindcss/tailwindcss/pull/520

Other than that this is awesome man! If you can revert this bit I'll pull it down and play with it then hopefully we can get it merged 👍 Thanks dude!

Reply:
Updated